bitcoin/src/taproot.rs

```diff
@@ -13,12 +13,13 @@ use bitcoin_internals::write_err;
 use secp256k1::{self, Scalar, Secp256k1};
 
 use crate::consensus::Encodable;
-use crate::crypto::key::{TapTweak, TweakedPublicKey, UntweakedPublicKey, XOnlyPublicKey};
 use crate::hashes::{sha256t_hash_newtype, Hash, HashEngine};
+use crate::crypto::key::{TapTweak, TweakedPublicKey, UntweakedPublicKey, XOnlyPublicKey};
```

Comment:
Nit: this looks like a mistake, since nothing changed, just ordering went from alphabetical to unknown.

Reply:
oh woops, hashes vs crypto you mean, right. Incoming rustfmt PRs will fix this.